src/redux/missions/missions.js

```diff
@@ -1,14 +1,57 @@
+/* eslint-disable no-case-declarations */
 import axios from 'axios';
 
 const GETMISSIONS = 'app/missions/MISSIONS_RETRIEVED';
+const JOINMISSION = 'app/missions/MISSION_JOINED';
+const LEAVEMISSION = 'app/missions/MISSION_LEFT';
```

Comment:
Please format the action types according to the ducks pattern.
Action types var name should match the actual action name.

Reply:
Done, please check